### PR TITLE
Hotfix/20191022 0947 issues 2176 2179

### PR DESCRIPTION
**Pull request by richard-jones**
Relates to:

https://github.com/DOAJ/doajPM/issues/2176
https://github.com/DOAJ/doajPM/issues/2179
https://github.com/DOAJ/doajPM/issues/2170

### CODE REVIEW

File: portality/static/css/doaj.css [hunk 1]
```diff
@@ -77,6 +77,14 @@ blockquote {
     background-color: white;
 }
 
+.navbar .nav .active > a:focus,             
+.navbar .nav > li:focus,
+.navbar .nav > li > a:focus,
+.navbar .navbar-header a:focus {
+    color: #F68B1F;
+    background-color: #eeeeee;
```
Comment (amdomanska) [inline portality/static/css/doaj.css]: It probably should be `background-color: white`. This way you will have the same background for :focus and for :hover

Comment (amdomanska) [reply portality/static/css/doaj.css]: Screenshot from 2019-10-23 10-22-18

Comment (richard-jones) [reply portality/static/css/doaj.css]: Yes, so, interesting point.  That was my first consideration, but an active nav is also highlighted white, so is there a usability question around having the tab-based navigation highlight in the same way as the current active nav?  That is, if you tab through the nav, it's not possible to visually determine which is the current nav and which is the focused nav item?

Having said that, the grey looks rubbish, and I notice that when you click a link it briefly becomes grey before becoming white.

Perhaps an additional signifier, like also underlining the focused one, as well as highlighting in white?

Comment (Steven-Eardley) [reply portality/static/css/doaj.css]: I agree, there should be different colours for focus and hover - you want navigating to the next item to be different to what you're currently on. The font colour change makes it readable and that's a good start.

Perhaps we want a lighter orange?

**Pull request closed without merging**